addons/point_of_sale/models/report_sale_details.py

```diff
@@ -62,7 +62,7 @@ def get_sale_details(self, date_start=False, date_stop=False, config_ids=False,
         else:
             config_currencies = self.env['pos.session'].search([('id', 'in', session_ids)]).mapped('config_id.currency_id')
         # If all the pos.config have the same currency, we can use it, else we use the company currency
-        if all(i == config_currencies.ids[0] for i in config_currencies.ids):
+        if config_currencies and all(i == config_currencies.ids[0] for i in config_currencies.ids):
```

Comment:
@robinengels but this line 65 check would still be true. And we still would have to check the config currencies.

Reply:
Hmm, yes you can keep it like that. But you can still add a test